add config item pessimistic-txn.constraint-check-in-place-pessimistic (#11939)
pingcap/docs-cn: system-variables.md

### `tidb_constraint_check_in_place_pessimistic` <span class="version-mark">从 v6.3.0 版本开始引入</span>

> - 作用域：SESSION

Reviewer: ```suggestion
- 作用域：SESSION
- 是否持久化到集群：否
```

Reviewer: @ekexium 请看看 “是否持久化到集群” 是 “否” 吗？

> - 默认值：当配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic) 为默认值 `true` 时，该变量的默认值为 `ON`。当配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic) 为 `false` 时，该变量的默认值为 `OFF`。

Reviewer: ```suggestion
- 默认值：当配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic-从-v640-版本开始引入) 为默认值 `true` 时，该变量的默认值为 `ON`。当配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic-从-v640-版本开始引入) 为 `false` 时，该变量的默认值为 `OFF`。
```

### `tidb_constraint_check_in_place_pessimistic` <span class="version-mark">从 v6.3.0 版本开始引入</span>

- 作用域：SESSION

> - 是否持久化到集群：是

Reviewer 2: ```suggestion
```
SESSION 变量不持久化到集群

> - 默认值：由配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic) 决定，该配置项的默认值为 `TRUE`(`ON`)。

Reviewer: ```suggestion
- 默认值：由配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic) 控制。当该配置项为默认值 `TRUE` 时，该变量的默认值为 `ON`。  
```

Reviewer: 该配置项为默认值 `TRUE`，这个 `TRUE` 可以小写吗，和 tidb-configuration-file.md 里保持一致

Author: 不写默认值的话，还要跳到配置项页面才能知道默认值是啥也太麻烦了……
on=true 这个应该找个地方集中写而不是每个碰到的地方都单独说一下吧……

Reviewer: 写了的😂。 on=true 这个集中写了，也需要用户跳到一个地方去看。
![image](https://user-images.githubusercontent.com/79440533/200487327-e61c37c1-e5f0-4683-8cdb-243931cb9ce5.png)

Author: 好吧。不过我觉得看起来很复杂
